packaging/installer/methods/kickstart.md

```diff
@@ -5,181 +5,130 @@ import TabItem from '@theme/TabItem';
+    - Build from source (last resort)
+4. Installs an auto-update cron job (unless disabled)
+5. Optionally connects your node to Netdata Cloud
 
-## Installation
+</details>  
 
-> **Tip**
->
-> If you are unsure whether you want nightly or stable releases, read the [related section](/docs/netdata-agent/versions-and-platforms.md) of our Documentation, detailing the pros and cons of each release type.
+---
 
-To install Netdata, run the following as your normal user:
+## Quick Overview
+
+| Task                  | Command / Location             | Notes                                   |
+|-----------------------|--------------------------------|-----------------------------------------|
+| Install Netdata       | Run `kickstart.sh`             | Choose nightly or stable release        |
+| Connect to Cloud      | Use claim token                | Connect node to Netdata Cloud           |
+| Customize install     | Pass flags to control behavior | Directory, release, update control      |
+| Export config for IaC | Copy config from Cloud UI      | For automation & Infrastructure as Code |
+
+---
+
+## Run the One-Line Install Command
+
+To install and connect to Netdata Cloud in a single step from your terminal:
 
 <Tabs>
   <TabItem value="wget" label="wget">
 
-  <OneLineInstallWget/>
+<OneLineInstallWget/>
 
   </TabItem>
   <TabItem value="curl" label="curl">
 
-  <OneLineInstallCurl/>
+<OneLineInstallCurl/>
 
   </TabItem>
 </Tabs>
 
-> **Note**
->
-> If you plan to also connect the node to Netdata Cloud, make sure to replace `YOUR_CLAIM_TOKEN` with the claim token of your space,
-> and `YOUR_ROOM_ID` with the ID of the Room you’re willing to connect the node to.
-
-## Optional parameters to alter your installation
-
-The `kickstart.sh` script accepts a number of optional parameters to control how the installation process works:
-
-### destination directory
-
-- `--install-prefix`
-  Specify a custom installation directory for local builds. If not provided, a default directory will be used based on your system.
-- `--old-install-prefix`
-  Specify the previous custom installation directory to be removed during the update process.
-
-### interactivity
-
-The script automatically detects if it is running interactively, on a user's terminal, or headless in a CI/CD environment. These are options related to overriding this behavior.
-
-- `--non-interactive` or `--dont-wait`
-  Don’t prompt for anything and assume yes whenever possible, overriding any automatic detection of an interactive run. Use this option when installing Netdata Agent with a provisioning tool or in CI/CD.
-- `--interactive`
-   Act as if running interactively, even if automatic detection indicates a run is non-interactive.
-
-### release channel
-
-By default, the script installs the nightly channel of Netdata, providing you with the most recent Netdata. For production systems where stability is more important than new features, we recommend using the stable channel.
+> [!TIP]
```

Comment:
```suggestion
> **Tip**
```

This does not render anything on Learn

![image](https://github.com/user-attachments/assets/96e0fb1a-e1a7-43bc-a7ca-c1cb42f70162)

Reply:
Please let's use [Admonitions](https://docusaurus.io/docs/2.x/markdown-features/admonitions). Forget about GH.

Reply:
agree